specification/compute/resource-manager/readme.md

```diff
@@ -231,6 +232,8 @@ These settings apply only when `--tag=package-2021-03-01-only` is specified on t
 input-file:
 - Microsoft.Compute/stable/2021-03-01/compute.json
 - Microsoft.Compute/stable/2021-03-01/runCommands.json
+- Microsoft.Compute/stable/2020-09-30/gallery.json
+- Microsoft.Compute/stable/2020-09-30/sharedGallery.json
```

Comment:
Should we add some swaggers of api-version `2020-09-30` to a `2021-03-01-only` tag?

Reply:
No, these should not be present. Please remove them.